README.md

```diff
@@ -9,6 +9,34 @@ FHIR Shorthand (FSH) is a specially-designed language for defining the content o
 
 For more information about the evolving FSH syntax see the [FHIR Shorthand Reference Manual](https://build.fhir.org/ig/HL7/fhir-shorthand/).
 
+## FHIR Foundation Project Statement
+
+### Maintainers
+
+This project is maintained by The MITRE Corporation.
+
+### Issues / Discussion
+
+For SUSHI issues, such as bug reports, comments, suggestions, questions, and feature requests, visit [SUSHI Github Issues](https://github.com/FHIR/sushi/issues).
```

Comment:
It turns out we're not _super_ consistent with this, but I think technically "GitHub" is the correct capitalization.

```suggestion
For SUSHI issues, such as bug reports, comments, suggestions, questions, and feature requests, visit [SUSHI GitHub Issues](https://github.com/FHIR/sushi/issues).
```